.github/.OwlBot.yaml

```diff
@@ -389,6 +391,8 @@ deep-copy-regex:
     dest: /dataplex/apiv1
   - source: /google/cloud/dataproc/v1/cloud.google.com/go/dataproc/apiv1
     dest: /dataproc/apiv1
```

Comment:
This could probably be removed since it will no longer be generated.

Reply:
Removed, thanks.